### PR TITLE
Add Jenkinsfile and initial files for ocp build

### PR DESCRIPTION
**Pull request by ghost**
Addition of Jenkins file that:
- Builds the code, runs tests 
- Runs basic static analysis (maven warnings, FIXME/TODO's)
- Builds a container image with the application (is discarded when the build is done)

Should demonstrate what is needed for basically getting a Jenkins job running. 

What it does not cover is to promote the container image, but adds the building blocks for enabling that bit later-on.

### CODE REVIEW

**Review by tokee — CHANGES_REQUESTED** [review]
Seems to skip unit testing?

File: kb-s2i/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# kb-s2i
+
```
Comment (tokee) [inline kb-s2i/README.md:2]: Write what all this is about without requiring the reader to know intrinsics of OpenShift and containers. A paragraph or two is fine.

Comment (tokee) [reply kb-s2i/README.md:2]: "The files in this folder is used for automatic builds, tests and image creation using OpenShift at the Royal Danish Library"

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+                properties([disableConcurrentBuilds()])
+
+                def mvnCmd = "mvn -s /etc/m2/settings.xml --batch-mode"
+
+                stage('checkout') {
+                    checkout scm
+                }
+
+                stage('Mvn clean package') {
+                    sh "${mvnCmd} clean package"
```
Comment (tokee) [inline Jenkinsfile]: `-PallTests`

**Review by tokee — CHANGES_REQUESTED** [review]

File: kb-s2i/README.md (hunk 1 above)
Comment (ghost) [reply kb-s2i/README.md:2]: Taken in

File: Jenkinsfile (hunk 2 above)
Comment (ghost) [reply Jenkinsfile]: Added

**Review by tokee — APPROVED** [review]
OJenkins output checked. Now runs tests.

**Pull request merged**